from src.app.exceptions.not_found_error import NotFoundError


class EndpointNotFoundError(NotFoundError):
    pass
